kinto_http/tests/test_batch.py

```diff
@@ -112,3 +112,25 @@ def test_batch_raises_exception_as_soon_as_subrequest_fails(self):
         with self.assertRaises(KintoException):
             batch.send()
         assert self.client.session.request.call_count == 1
+
+    def test_results_attribute_is_available(self):
```

Comment:
I'm not sure if we have both. Especially because this `results` attributes is chunked. I know we'd lose the headers information, but I would rather make it private. Especially if we don't make it official in docs.